biojava-integrationtest/src/test/java/org/biojava/nbio/structure/test/StructureTest.java

```diff
@@ -119,13 +119,13 @@ public void testReadPDBFile() throws Exception {
 		assertEquals(64, c2.getAtomGroups(GroupType.HETATM).size());
 		assertEquals(0, c2.getAtomGroups(GroupType.NUCLEOTIDE).size());
 
-		List<Compound> compounds= structure.getCompounds();
+		List<EntityInfo> compounds= structure.getEntityInformation();
```

Comment:
Shouldn't this be structure.getEntityInfo() ? getter and setter methods should use the same names as the classes, to make it easier to work with Reflection.